enclave_manager/avalon_enclave_manager/kme/kme_enclave_manager.py

```diff
@@ -43,6 +43,19 @@ def __init__(self, config):
 
         super().__init__(config)
 
+# -------------------------------------------------------------------------
+
+    def _create_signup_data(self):
+        """
+        Create KME signup data.
+
+        Returns :
+            signup_data - Relevant signup data to be used for requests to the
```

Comment:
For doxygen, prefix signup_data with @returns

Reply:
Updated all function comments in enclave info modules for doxygen. Will take up that for all the managers in a separate PR owing to the bulk of this one already.